### ISSUE TITLE
Implement a provider for the package manager

### ISSUE THREAD

**Issue by davidanthoff** [davidanthoff/Query.jl#51]
Would be fun if one could easily query the package manager database with Query.

**Comment by ararslan** [comment]
This would indeed be fun! I don't know how generally useful it would be to users, though, and at least on my machine, the code I wrote that constructs the SQLite database takes a rather annoying amount of time.

**Comment by davidanthoff** [comment]
Yeah, it is pretty slow, takes about 30 seconds to read the data... Maybe it should actually live in PkgDev, package developers are probably the ones that might use this.

**Comment by davidanthoff** [comment]
I'm adding an example in #110 that shows how to do this. The package db code all comes from https://github.com/JuliaPackaging/MetadataTools.jl.
